acceptance/setup/aio/pre-suite/010_Install.rb

```diff
@@ -44,3 +46,25 @@
     end
   end
 end
+
+step 'Install build dependencies on master' do
+
+ MASTER_PACKAGES = {
+    :redhat => [
+      'git',
+      'java-1.8.0-openjdk-devel',
+    ],
+  }
+  install_packages_on(master, MASTER_PACKAGES, :check_if_exists => true)
+end
+
+step 'Confirm SSH Forwarding Enabled on master' do
```

Comment:
Do you mean the ssh forwarding check starting at line 61? I believe it makes the ssh auth available to the user executing beaker also available to the user on the test machine when beaker has ssh'ed onto it.

Looking at it, I'm not sure if we should be testing it every run; or if we do test it, it should be as early as possible in the setup so we fail faster